deployments/managed-services.mdx

```diff
@@ -0,0 +1,9 @@
+---
+title: "Managed services"
+---
+
```

Comment:
@mtgoncalves1 before we get into the content of our managed services, we should start with an overview of the dependency graph - this will help contextualize what we're talking about, especially as we start talking about the nodes and other things that are specific to our product.

From there, we can break it down into further sections of what we're adding (just the trafik + Cert manager).